umbraco/Umbraco-CMS
# Fix Rollback feature not creating diffs when properties are moved between tabs in EditorModel events

The rollback feature breaks if you change which tab a property is on using the EditorModel events as it can't generate a diff.
This is because the createDiff function assumes the old version and the current version have the same tab/property indexes.
This PR fixes this by finding the old property by the current property alias.

### Prerequisites


If there's an existing issue for this PR then this fixes 

How to test:
Reorder properties on tabs in  the EditorModelEvent.

Create two versions of a document.
Try to roll back to the previous version
Rollback diff won't appear (just shows loading)
Unable to rollback.
